samples/contrib/aws-samples/titanic-survival-prediction/titanic-survival-prediction.py

```diff
@@ -6,33 +6,41 @@
 from kfp import gcp
```

Comment:
This is only formatting. All of our samples and code is formatted the same way. We have a check in place to ensure everything is linted correctly and this would be the only exception?